modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/ObjectEntriesFDSPropsTransformer.tsx

```diff
@@ -3,24 +3,75 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-interface ObjecEntriesDataRendererProps {
+import {openModal} from 'frontend-js-web';
+
+import ModalDeleteObjectEntry from '../ObjectEntry/ModalDeleteObjectEntry';
+
+interface ItemData {
```

Comment:
@Artartur Do not replicate interfaces. Move the other one you've created to the index.d.ts file and call it in both places.

Reply:
Done!